controllers/storagecluster/cephblockpools.go

```diff
@@ -118,13 +119,20 @@ func (o *ocsCephBlockPools) reconcileMgrCephBlockPool(r *StorageClusterReconcile
 	_, err = ctrl.CreateOrUpdate(r.ctx, r.Client, cephBlockPool, func() error {
 		cephBlockPool.Spec.Name = ".mgr"
 
-		// Pass the Replicated Size Spec for the default CephBlockPool from the storageCluster CR
+		setDefaultMetadataPoolSpec(&cephBlockPool.Spec.PoolSpec, storageCluster)
+		// Pass the Replicated Size Spec for the default cephBlockPool spec if specified
 		manageCBPSpec := &storageCluster.Spec.ManagedResources.CephBlockPools
 		if manageCBPSpec.PoolSpec != nil && manageCBPSpec.PoolSpec.Replicated.Size != 0 {
```

Comment:
If the blockpool is using erasure coding, the .mgr and .nfs pools will still get replica 3 from the `setDefaultMetadataPoolSpec()` method, right?

Reply:
Yes they should.